pkg/base/test_server_args.go

```diff
@@ -128,9 +128,6 @@ type TestServerArgs struct {
 
 	// IF set, the demo login endpoint will be enabled.
 	EnableDemoLoginEndpoint bool
-
-	// If set, testing specific descriptor validation will be disabled. even if the server
-	DisableTestingDescriptorValidation bool
```

Comment:
Note: I'm effectively reverting https://github.com/cockroachdb/cockroach/commit/dd9043993e47a385556da0ec4450740128f0810d, that framework is not actually used in any way.